src/main/java/com/etsy/statsd/profiler/server/RequestHandler.java

```diff
@@ -18,32 +21,61 @@
  * @author Andrew Johnson
  */
 public class RequestHandler {
-
+    
     /**
      * Construct a RouteMatcher for the supported routes
      *
      * @param activeProfilers The active profilers
      * @return A RouteMatcher that matches all supported routes
      */
-    public static RouteMatcher getMatcher(final Map<String, ScheduledFuture<?>> activeProfilers) {
+    public static RouteMatcher getMatcher(final Map<String, ScheduledFuture<?>> runningProfilers,  Map<String, Profiler> activeProfilers, AtomicReference<Boolean> isRunning, LinkedList<String> errors) {
         RouteMatcher matcher = new RouteMatcher();
-        matcher.get("/profilers", RequestHandler.handleGetProfilers(activeProfilers));
-        matcher.get("/disable/:profiler", RequestHandler.handleDisableProfiler(activeProfilers));
-
+        matcher.get("/profilers", RequestHandler.handleGetProfilers(runningProfilers));
+        matcher.get("/disable/:profiler", RequestHandler.handleDisableProfiler(runningProfilers));
+        matcher.get("/status/profiler/:profiler", RequestHandler.handleProfilerStatus(activeProfilers));
```

Comment:
Could you document the new endpoints with the others [here](https://github.com/etsy/statsd-jvm-profiler#embedded-http-server)?